fplll/bkz.cpp

```diff
@@ -499,6 +493,18 @@ template <class FT> bool BKZReduction<FT>::bkz()
   cputime_start = cputime();
 
   m.discoverAllRows();
+  
+  if (sld)
+  {
+    m.updateGSO();
+    sld_potential = m.get_slide_potential(0, num_rows, param.block_size);
+  }
+  
+  // the following is necessary, since sd-bkz starts with a dual tour and 
+  // svp_reduction calls size_reduction, which needs to be preceeded by a 
+  // call to lll lower blocks to avoid seg faults
+  if (sd)
+    lll_obj.lll(0, 0, num_rows);
```

Comment:
I'm leaning towards unconditional calling regardless, just to ensure a sane state, but perhaps people really really want BKZ_NO_LLL so they shall have it. Merge time.